packages/dds/shared-object-base/src/remoteObjectHandle.ts

```diff
@@ -52,7 +48,7 @@ export class RemoteFluidObjectHandle extends FluidHandleBase<FluidObject> {
 			};
 			this.objectP = this.routeContext.resolveHandle(request).then<FluidObject>((response) => {
 				if (response.mimeType === "fluid/object") {
-					const fluidObject: FluidObject = response.value;
+					const fluidObject: FluidObject = response.value as FluidObject;
```

Comment:
Any validation we can do here?

Reply:
Not familiar with the space, but I think the existing check for mimeType is enough?